docs/guides/javascript.md

```diff
@@ -42,17 +26,27 @@ spectral.addFunctions(oas3Functions());
 oas3Rules()
   .then(rules => spectral.addRules(rules))
   .then(() => {
+    // optional: extend with your own custom rules
     spectral.addRules({
-      // .. extend with your own custom rules
+      // ... snip
     });
 
-    // run!
-    spectral.run(myOAS).then(results => {
-      console.log(JSON.stringify(results, null, 4));
-    });
+    spectral.run(myOpenApiDocument).then(results => console.log(results);
   });
 ```
 
+_**Note:** Spectral v4.0 does not yet support loading rulesets in the same way as the CLI, so this code is a little bloated adding functions and rules seperately. This example will become much nicer in v4.1 when we add loading custom rulesets. 
```

Comment:
Are we adding it in 4.1 100%?

Reply:
Maybe, rather than anticipate it, we could update docs with the final content when we actually ship this feature?

Reply:
Yes. We are adding it in v4.1. Again this feels like an oversight if unmentioned, so we want to let a casual reader know we are on top of it. I don't always share roadmap in docs because it sets expectations, but we have comitted to shipping this feature in v4.1 and I think its important to mention its coming.